Week-5/index.js

```diff
@@ -0,0 +1,34 @@
+// // Write a function called vowelCount which accepts a string and returns a map
+// // where the keys are numbers and the values are the count of the vowels in the string
+
+let givenString = 'anyAlikeChar';
+
+function isVowel(char) {
+	return 'aeiou'.includes(char);
+}
+
+function vowelCount(str) {
+	const map = new Map();
+	str = str.toLowerCase();
+
+	for (let char of str) {
+		/**
+		 * Logic
+		 *
+		 * First check if the character is vowels or not,
+		 * if not do nothing,
+		 * else check if that character exists in the map, if not add char with value 1
+		 * otherwise increase the count of char
+		 */
+
+		isVowel(char)
```

Comment:
Rather than checking for vowel , you can maintain the map with default values as 

{'a': 0, 'e': 0, 'i': 0, 'o': 0, 'u': 0}

so that isVowel is not needed , you can check like map.has(char)